ui/src/containers/BeamlineSetupContainer.jsx

```diff
@@ -11,7 +11,7 @@ import DeviceState from '../components/DeviceState/DeviceState';
 import LabeledValue from '../components/LabeledValue/LabeledValue';
 import MachInfo from '../components/MachInfo/MachInfo';
 import OneAxisTranslationControl from '../components/MotorInput/OneAxisTranslationControl';
-import * as SampleViewActions from '../actions/sampleview';
+import * as sampleViewActions from '../actions/sampleview'; // eslint-disable-line import/no-namespace
```

Comment:
[no-namespace](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-namespace.md)

Ideally, all the actions should be imported one by one to make it easier to navigate through the codebase and know where each action is actually used. Some lower-level components should probably connect to the store directly. Anyway, it's not crucial, and since this is such a deep change, I think disabling the linting rule is satisfactory.